app/code/Magento/Theme/view/frontend/web/css/validate.css

```diff
@@ -1,20 +0,0 @@
-/**
- * Copyright © Magento, Inc. All rights reserved.
- * See COPYING.txt for license details.
- */
-input.mage-error, textarea.mage-error, ul.mage-error, select.mage-error {
-    background: none repeat scroll 0 0 #FAEBE7 !important;
-    border: 1px dashed #EB340A !important;
-
-}
-div.mage-error {
-    background: url("../images/validation_advice_bg.gif") no-repeat scroll 2px 1px transparent;
```

Comment:
if that's not needed - pls remove the image aswell

Reply:
Will do it most likely separately, as there are some additional images that are left from M1.